intuita-webview/src/main/TreeView/index.tsx

```diff
@@ -83,11 +94,13 @@ const TreeView = ({ node }: Props) => {
 				id={node.id}
 				label={node.label ?? ''}
 				icon={icon}
-				indent={depth * 30}
+				indent={depth * 12}
```

Comment:
what is the unit of it? `px`? `rem`?

Reply:
in px. We do `<div style={{ minWidth: `${indent}px` }} />` when building `TreeItem`.